.goreleaser.yaml

```diff
@@ -92,16 +93,17 @@ dockers:
       - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
       - "--label=org.opencontainers.image.documentation=https://aquasecurity.github.io/starboard/v{{ .Version }}/"
       - "--platform=linux/amd64"
+  
   - image_templates:
-      - "docker.io/aquasec/starboard-operator:{{ .Version }}-amd64"
+      - "docker.io/aquasec/starboard-operator:ubi8-{{ .Version }}-amd64"
```

Comment:
Why are you removing Alpine-based image? I think we should keep publishing Alpine-based images and add UBI-based images.

Reply:
That's okey. Alpine images don't have suffixes